### PR TITLE
fix(data-warehouse): change data warehouse person prop format

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- colon not being used

## Changes

- use colon instead of period

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

File: frontend/src/scenes/data-warehouse/external/dataWarehouseJoinsLogic.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const dataWarehouseJoinsLogic = kea<dataWarehouseJoinsLogicType>([
                     if (table) {
                         acc.push(
                             ...Object.values(table.fields).map((column) => ({
-                                id: `${join.field_name}.${column.name}`,
+                                id: `${join.field_name}: ${column.name}`,
```
Comment (Gilbert09) [inline frontend/src/scenes/data-warehouse/external/dataWarehouseJoinsLogic.ts:49]: We split by `.` elsewhere too - so I'd recommend keeping it as a dot separator and change the `property_to_expr`

Comment (Gilbert09) [reply frontend/src/scenes/data-warehouse/external/dataWarehouseJoinsLogic.ts:49]: https://github.com/PostHog/posthog/pull/22685/files#diff-9e284b4a4f022ffe99ed273e872c3f351daf79c73b4964860abf92300bed185fR32

Comment (Gilbert09) [reply frontend/src/scenes/data-warehouse/external/dataWarehouseJoinsLogic.ts:49]: https://github.com/PostHog/posthog/pull/22685/files#diff-66a83f26c784f7721e4df1189a3cf662dd5dac9319aa15a60c0a8ad781d84ccfR191

Comment (Gilbert09) [reply frontend/src/scenes/data-warehouse/external/dataWarehouseJoinsLogic.ts:49]: (or change the above 2 files to be colons, but I do think a dot is likely better)

Comment (EDsCODE) [reply frontend/src/scenes/data-warehouse/external/dataWarehouseJoinsLogic.ts]: changed to dot! and updated test

**Review by Gilbert09 — APPROVED** [review]

**Pull request merged**